src/views/Write.vue

```diff
@@ -41,11 +42,11 @@ export default {
   },
   methods: {
     async savePost (newArticle) {
-      const { title, content, boardId, attachments } = newArticle
+      const { title, boardId, attachments } = newArticle
       if (title === '') {
         this.isTitleEmpty = true
       }
-      if (content === '{"type":"doc","content":[{"type":"paragraph"}]}') {
+      if (this.$refs.write.getContent() === '{"type":"doc","content":[{"type":"paragraph"}]}') {
```

Comment:
```suggestion
      if (this.$refs.write.getContent() === '<p></p>') {
```
아마도 이런식으로 수정해야 할 것 같은데 (html 출력으로 바뀌었으므로) 조건 확인 한 번만 부탁드려요.

Reply:
`<p></p>` 로 바꾸니 잘 작동하는 거 확인했습니다! 감사합니다 :D